app/Http/Controllers/AdminDash.php

```diff
@@ -1390,6 +1390,14 @@ public function saveNewEvent(Request $request) {
             'banner_url' => 'nullable|url'
         ]);
 
+        $valid_time_expr = '/^(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]/';
+        if (!preg_match($valid_time_expr, $request->start_time)) {
```

Comment:
Can this not be added as validation in that function? I think there's a date time validation that we can do

Reply:
Fixed - thank you, that was a great suggestion!